modules_css/form_builder.css

```diff
@@ -520,10 +520,10 @@ div.form-builder-new-field span.progress {
 #form-builder-field-palette ul li.field-city a {
   background-position: 5px -1362px;
 }
-#form-builder-field-palette ul li.field-direct_mail_newsletter a {
-  background-position: 5px -84px;
-}
-#form-builder-field-palette ul li.field-email_newsletter a, #form-builder-field-palette ul li.field-newsletter a {
+#form-builder-field-palette ul li.field-email_newsletter a,
+#form-builder-field-palette ul li.field-newsletter a,
+#form-builder-field-palette ul li.field-phone_opt_in a,
+#form-builder-field-palette ul li.field-post_opt_in a {
```

Comment:
generic opt-in field is missing: `#form-builder-field-palette ul li.field-opt_in a`

Reply:
The generic opt-in field should be hidden in the palette, but I does no harm to include it either.